pkg/controllers/kube-apiserver.go

```diff
@@ -298,6 +304,20 @@ func (s *KubeAPIServer) Run(ctx context.Context, ready chan<- struct{}, stopped
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	if !s.skipLoInterface {
```

Comment:
It's not clear where to draw a line for the single responsibility principle here. Should this IP management be done inside the KubeAPIServer service or can it be its own thing that KubeAPIServer depends on?

Reply:
Turned this into a new service. Makes more sense, thanks for the suggestion.